samples/tab-device-permissions/nodejs/src/components/CaptureVideo.tsx

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { useEffect, useState } from 'react';
+import * as microsoftTeams from "@microsoft/teams-js";
+import { Card, Flex, Text, Button, CardHeader, CardBody, Video } from '@fluentui/react-northstar'
+
+/**
+ * The 'CaptureVideo' component
+ * of your app.
+ */
```

Comment:
We can add a single line comment like below:
// The 'CaptureVideo' component of your app.